lib/generators/tcramer/install_generator.rb

```diff
@@ -11,7 +11,7 @@ def write_your_first_rake_task
         %Q{
 desc 'Manage all the things'
 task :manage do
-  puts Tcramer::ISMS.sample
+  puts Zalgo.he_comes(Tcramer::ISMS.sample)
```

Comment:
do you intend to unconditionally zalgo the management? it becomes predictable, and then who's keeping your team on their toes?

Reply:
@barmintor I considered this and then failed to give this work the proper time; I concur that zalgoization should not be applied to every management action, and should instead be used sparingly.

I'm going to add a test suite, get the builds passing in CI, and then do some rearchitecting before revisiting this PR. Thank you for the constructive feedback!